projects/igniteui-angular/src/lib/banner/banner.component.html

```diff
@@ -0,0 +1,27 @@
+<igx-expansion-panel #expansionPanel [animationSettings]="animationSettings" (onCollapsed)="onExpansionPanelClose()" (onExpanded)="onExpansionPanelOpen()"
+    [collapsed]="collapsed">
+    <igx-expansion-panel-body>
+        <div class="igx-banner">
+            <div class="igx-banner__message">
+                <div *ngIf="bannerIcon" class="igx-banner__illustration">
```

Comment:
@desig9stein @simeonoff Could you take a look at the refactored template, when you get a chance?
Can the `igx-banner__illustration` class wrapper be removed? We don't need it to set a gray circle background for the image and that could simplify the template.

Reply:
@ViktorSlavov Unfortunately the class is used to add other styles apart from setting a shape and background. @desig9stein Could you please remove the oval and the background?